### PR TITLE
Animations Added to SDL

### PR DESCRIPTION
**Pull request by csharpee**
Added updated animations from @UrgeyBergy
Also included updates for "midnight" theme integration once integrated via #109

### CODE REVIEW

File: src/controller.cpp [hunk 1]
```diff
@@ -1523,7 +1523,7 @@ void Controller::shutdownhushd()
         QDialog d(main);
         Ui_ConnectionDialog connD;
         connD.setupUi(&d);
-        connD.topIcon->setBasePixmap(QIcon(":/icons/res/icon.ico").pixmap(256, 256));
+        connD.topIcon->setPixmap(QIcon(":/icons/res/icon.ico").pixmap(256, 256));
```
Comment (leto) [inline src/controller.cpp:1526]: Why is this change needed?

Comment (csharpee) [reply src/controller.cpp:1526]: In the **connection.ui** form, I had to change topIcon to a QLabel class for using animations; it didn't show animation without that change. 

By doing so, produced the QLabel error for setBasePixmap. 
`no member named 'setBasePixmap' in QLabel`

After the UI changes were created, the build error was produced via **./build.sh**: 

```
void Controller::shutdownhushd()’:
src/controller.cpp:1526:24: error: ‘class QLabel’ has no member named ‘setBasePixmap’; did you mean ‘setPixmap’?
 1526 |         connD.topIcon->setBasePixmap(QIcon(":/icons/res/icon.ico").pixmap(256, 256));
      |                        ^~~~~~~~~~~~~
      |                        setPixmap
make: *** [Makefile:3645: bin/controller.o] Error 1
make: *** Waiting for unfinished jobs....
```
recommended me to use **setPixmap**.

**Comment by csharpee** [comment]
Okay, good to know for future commits @duke
Last night I dived deep into finding a solution to why the black / white logic animations where not working; reason being all the spacing checks lol.

**Comment by csharpee** [comment]
Did you want me to redo the changes? I am able to do so if it would make things easier.

**Comment by leto** [comment]
@csharpee it's up to @DenioD to merge this, I have no objections

**Comment by DenioD** [comment]
Thanks @csharpee :-)
